contributing.md

```diff
@@ -40,3 +40,17 @@ Using karma (real browsers will get launched).
 ```bash
 yarn test
 ```
+
+## IDEs
```

Comment:
This is about adding general flowtype to vscode ... how is this related to JSS project?

I think we should add documentation that specifically helps with JSS related problems

Reply:
One doesn't even need ide integration to contribute to JSS, one can run flow over console ...